### PR TITLE
feat: add GitHub section component

### PR DESCRIPTION
**Pull request by burak-balci**

### CODE REVIEW

**Comment by omeersari** [comment]
@burak-balci @eraygundogmus Responsive design için pek uygun gelmedi ama responsive design tasarımı da yok. Bunu daha sonra mı yazacağız ?

**Review by muratdemirci — APPROVED** [review]
LGTM 🤌

**Comment by eraygundogmus** [comment]
@burak-balci Hey, Burak! Could you please update your PR by adding responsive designs?

File: components/GitHubSectionCard/GitHubSectionCard.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      <div className={styles.innerContainer}>
+        <div className={styles.titleContainer}>
+          <Icon icon="star" size={20} />
+          <span className={styles.title}></span>Frontendship
+        </div>
+        <div className={styles.images}>
+          <div className={styles.imageContainer}>
+            <img
+              src="https://images.pexels.com/photos/762080/pexels-photo-762080.jpeg?auto=compress&cs=tinysrgb&w=600"
+              alt=""
```
Comment (OrhanOzkercin) [inline components/GitHubSectionCard/GitHubSectionCard.tsx]: We may want to add alt tag to images

**Review by OrhanOzkercin — CHANGES_REQUESTED** [review]
Could you please sync `package-lock.json` as far as I see there is no difference. So it should not be shown as a difference.

**Comment by eraygundogmus** [comment]
LGTM!

**Review by eraygundogmus — APPROVED** [review]

**Pull request merged**